tests/region/enc/enc.c

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <assert.h>
+#include <openenclave/bits/sgx/region.h>
```

Comment:
If this test is inherently SGX-specific then it should have sgx in the directory name or filename.